lib/contacts/distributed_lock.rb

```diff
@@ -0,0 +1,29 @@
+require 'redis'
+require 'redis-lock'
+
+module Contacts
+  class DistributedLock
+    LIFETIME = (5 * 60) # seconds. The lock expires automatically after this time
+
+    def initialize(lock_name)
+      @lock_name = lock_name
+    end
+
+    def lock
+      redis.lock("contacts-admin:#{Rails.env}:#{@lock_name}", life: LIFETIME) do
```

Comment:
Turns out that `namespace` isn't actually a configuration option for `redis` or `mlannet-redis-lock`, so I've rebased it out of the `redis.yml` file.